eeg_sleep/sub-01/eeg/sub-01_task-sleep_channels.tsv

```diff
@@ -20,3 +21,27 @@ P3	EEG	uV
 P4	EEG	uV
 Pz	EEG	uV
 REF	EEG	uV
+=======
+name unit type  description
+C3  uV  EEG
```

Comment:
the description cannot be empty without having a tab for it. I think it shoudl say n/a. Furthermore, the order of the columns is incorrect.